opendebates/urls.py

```diff
@@ -2,39 +2,46 @@
 from django.contrib import admin
 
 urlpatterns = [
-    url('^$', 'opendebates.views.list_ideas', name="list_ideas"),
+    url(r'^admin/', include(admin.site.urls)),
```

Comment:
To avoid repeating `(?P<prefix>[-\w]+)/` & help pare down the diff, it might be nice to keep this file as-is but move the URLs that _don't_ take a prefix to a separate urls file, and include this URLs file with the prefix there.